src/AB1805_RK.h

```diff
@@ -74,8 +74,11 @@ class AB1805 {
      * 
      * The FOUT/nIRQ pin is also used for one-time and periodic interrupts.
      */
-    AB1805 &withFOUT(pin_t pin) { foutPin = pin; return *this; };
-
+    AB1805 &withFOUT(pin_t pin) { 
+        foutPin = pin; 
+        pinMode(foutPin, OUTPUT);
```

Comment:
Do we not already set the pin as an output in our pinModes or in other firmwares at the beginning of setup?